src/posix/platform/multicast_routing.cpp

```diff
@@ -63,7 +63,7 @@ void MulticastRoutingManager::Init(otInstance *aInstance)
 void MulticastRoutingManager::Deinit(void)
 {
     Mainloop::Manager::Get().Remove(*this);
-    otBackboneRouterSetMulticastListenerCallback(mInstance, nullptr, nullptr);
```

Comment:
Wonder why `otBackboneRouterSetMulticastListenerCallback(mInstance, nullptr, nullptr)` is unnecessary. @wgtdkp